src/components/mainPage/CategoryBtn.module.scss

```
@@ -40,3 +52,23 @@
   font-weight: 600;
   color: #0d0d0d;
 }
+
+
+@media (max-width: 390px) {
```

각자 휴대폰에서 실행해보니 400px 설정이 적합한거 같더라고요!

```suggestion
@media (max-width: 400px) {
```

---

네! 400으로 변경하겠습니다